app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Forgot your password?</h2>
```

Comment:
The user is unable to reset the password in case he forgets it. Kindly see the error below which occurs after entering the email to reset the password.
![forget](https://user-images.githubusercontent.com/35504149/157260032-105e26a8-89f8-4970-881b-1feea7c8f66b.png)